src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    public void doWork() {
+        System.out.println("Bulldozer started its work.");
+    }
+
+    public void stopWork() {
+        System.out.println("Bulldozer stopped working.");
```

Comment:
Similarly, the stop message should be more informative. Include the class name to clarify which machine stopped, e.g., 'Bulldozer has stopped working.'